src/ci/docker/host-x86_64/x86_64-gnu-llvm-18/Dockerfile

```diff
@@ -59,5 +59,9 @@ COPY host-x86_64/dist-x86_64-linux/build-gccjit.sh /scripts/
 
 RUN /scripts/build-gccjit.sh /scripts
 
-COPY scripts/x86_64-gnu-llvm.sh /tmp/script.sh
-ENV SCRIPT /tmp/script.sh
+ARG SCRIPT_ARG
+COPY scripts/x86_64-gnu-llvm.sh /tmp/x86_64-gnu-llvm.sh
+COPY scripts/x86_64-gnu-llvm1.sh /tmp/x86_64-gnu-llvm1.sh
+COPY scripts/x86_64-gnu-llvm2.sh /tmp/x86_64-gnu-llvm2.sh
+COPY scripts/x86_64-gnu-llvm3.sh /tmp/x86_64-gnu-llvm3.sh
```

Comment:
I can't only copy the right file because this [line](https://github.com/MarcoIeni/rust/blob/2bf1cec41b457bae459d7f18168e147fb467d898/src/ci/docker/run.sh#L64) wants the exact path (I tried to use ${SCRIPT_ARG} and it failed.